### PR TITLE
Input cost tag into RouteCard

### PR DESCRIPTION
**Pull request by CarlOllington**

### CODE REVIEW

File: src/components/RouteCard/index.jsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ function RouteCard({journeyInfo}) {
                     <h2>To: {journeyInfo[0].to}</h2>
                 </div>
                 <h2>Line: {(journeyInfo[0].line.replaceAll('_', ' '))}</h2>
+                <h2>Cost: {"£"+journeyInfo[0].cost}</h2>
```
Comment (Henry-Tottle) [inline src/components/RouteCard/index.jsx]: Could probably move the £ out of the {}

Comment (matt-ioa) [reply src/components/RouteCard/index.jsx]: You absolutely should!

**Review by matt-ioa — CHANGES_REQUESTED** [review]
Just one small change

**Review by matt-ioa — APPROVED** [review]
Well done!

**Pull request merged**